src/Providers/CountriesServiceProvider.php

```diff
@@ -22,7 +24,7 @@ class CountriesServiceProvider extends ServiceProvider
     public function register()
     {
         /** @var \Illuminate\Config\Repository $config */
-        $config = $this->app['config'];
+        $config = $this->app->config;
```

Comment:
Are you 100% sure this is right? `$this->app['config']` will get the right binding, I'm not sure what the property `$config` is for...

Reply:
Yep, I've checked and this looks to be just another way of referencing exactly the same thing.